src/game/phase.cpp

```diff
@@ -56,6 +60,9 @@ Phase* BuildPhase::update() {
 
 void BuildPhase::teardown() {
     joystick_conn.disconnect();
+    button_conn.disconnect();
+
+    events::build::end_build_event();
```

Comment:
Sure, it's okay for now.  Later on, the server would determine when build phase ends, and should broadcast a message indicating that it has ended.